packages/cli/src/commands/setup/vite/viteHandler.js

```diff
@@ -0,0 +1,136 @@
+import fs from 'fs'
+import path from 'path'
+
+import chalk from 'chalk'
+import { Listr } from 'listr2'
+
+import { addWebPackages } from '@redwoodjs/cli-helpers'
+import { getConfigPath } from '@redwoodjs/internal/dist/paths'
+import { errorTelemetry } from '@redwoodjs/telemetry'
+
+import { getPaths, transformTSToJS, writeFile } from '../../../lib'
+import c from '../../../lib/colors'
+import { isTypeScriptProject } from '../../../lib/project'
+
+const { version } = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, '../../../../package.json'), 'utf-8')
+)
```

Comment:
Since the diff isn't helpful, here's the first part of the addition to the handler to fix which version is installed. It's the same as the auth providers:

https://github.com/redwoodjs/redwood/blob/a3e9480d2ae53afe1665d608cd7a49fd025f9fe0/packages/auth-providers/auth0/setup/src/setupHandler.ts#L8-L10